### DISCUSSION TITLE
NEXT: checkboxes specific style rule not included

### DISCUSSION THREAD

**Discussion by SIMULATAN** [skeletonlabs/skeleton#2765] [Q&A] [answered]
I am trying to use the `checkbox` class to have styled toggles.
However, the visual appearance of my checkboxes doesn't change.

After tons of debugging, I've found that there's a few important rules missing, most notably `appearance: none`.
My site:
![site screenshot](https://github.com/user-attachments/assets/107a91a4-9f35-4d85-b842-b5b901eb877c)
Docs site:
![docs screenshot](https://github.com/user-attachments/assets/bfa1e671-c362-4e2e-8cc1-c6abe0fbba96)

Are there any global styles I forgot to include somehow? I made sure to follow the setup exactly and use the latest versions (`@skeletonlabs/skeleton 3.0.0-next.2` (upgraded from .1 after encountering this issue to test if it was fixed) and `@skeletonlabs/skeleton-svelte 1.0.0-next.3` with svelte `5.0.0-next.188`)

**Comment by endigo9740** [comment, marked as answer]
@SIMULATAN it's not yet indicated on the v3 docs yet, but form elements have the same prerequisite of Tailwind's Form plugin described here in Skeleton v2. This acts as a reset to normalize the styles before our custom styles are applied:

https://www.skeleton.dev/elements/forms#install-the-forms-plugin

If you could please confirm this resolves the issue for you, and we'll aim to get this information added to the doc site.

**Reply by SIMULATAN** [reply]
Wow, that was a rapid answer!

![fixed checkbox](https://github.com/user-attachments/assets/c8221d60-28c0-43da-ad20-60fe8bc7fd9a)
It sure does fix the problem :partying_face: Thank you so much!

If you want to, I can create a PR for the next site and add the install section.

**Reply by endigo9740** [reply]
Glad to help!

> If you want to, I can create a PR for the next site and add the install section.

We always welcome contributions. That would be very helpful. We have contribution tips and guidelines here if it helps:
https://next.skeleton.dev/docs/resources/contribute

I'd recommend you start by creating a ticket here on GitHub so we can track this.

Otherwise feel free to ping me here or on Discord if you need support. The team and I do try to answer asap! :)
